use-cases/client-server-msgs-perf/svmsg_file_server.c

```diff
@@ -618,4 +630,24 @@ printSummaryStats(const char *run_descr, Client_info *clients, unsigned int num_
            time_metric_name(clock_id), (elapsed_ns / num_ops),
            svr_throughput, value_str(svr_throughput),
            cli_throughput, value_str(cli_throughput));
+
+    // Generate one-line summary for post-processing by perf-report.py
+    if (outfile) {
+        printf("tail -f %s\n", outfile);
+        FILE *fh = fopen(outfile, "a");
+        if (!fh) {
+            printf("Error! Unable to open perf-metrics output file '%s\n",
+                   outfile);
+            exit(EXIT_FAILURE);
+        }
+        fprintf(fh, "%s, NumClients=%u, NumOps=%" PRIu64 " (%s)"
+                    ", Server throughput=%lu (%s) ops/sec"
+                    ", Client throughput=%lu (%s) ops/sec"
+                    ", elapsed_ns=%lu (%s) ns\n",
+                run_descr, num_clients, num_ops, value_str(num_ops),
```

Comment:
The format of the one-line summary message is verified by new pytests which cross-check Python parsing logic.